build/NuGetPackages/Microsoft.Build.Tasks.Core.nuspec

```diff
@@ -67,6 +67,7 @@
         <dependency id="Microsoft.Build.Utilities.Core" version="[$version$]" />
         <dependency id="System.Collections.Immutable" version="1.3.1" />
         <dependency id="System.Reflection.Metadata" version="1.3.0" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" />
```

Comment:
devenv.exe.config has a binding redirect for lower versions of this:

```xml
<dependentAssembly>
  <assemblyIdentity name="System.Runtime.InteropServices.RuntimeInformation" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
  <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
</dependentAssembly>
```

Since this is outside of that range we can still ship and use the newer one, but we need to be a bit careful about it. Do we need this version specifically?